src/main/java/cam72cam/immersiverailroading/thirdparty/opencomputers/AugmentDriver.java

```diff
@@ -236,6 +237,44 @@ private FluidStack getFluid() {
 				}
 				info.put("direction", dir.toString());
 				
+				for (EntityCoupleableRollingStock car : stock.getTrain()) {
+					if (car instanceof Locomotive) {
+
+						EntityRollingStockDefinition def = car.getDefinition();
+
+						info.put("LOCO_"+locomotiveCount+"_id", def.defID);
```

Comment:
Instead of copying and pasting this code from above, it should be broken out into a common function and added as a List of Dicts to the info dict.

Reply:
Ex:
```
List<Object> locos = new ArrayList<Object>();
for (EntityCoupleableRollingStock car : stock.getTrain()) {
    if (car instanceof Locomotive) {
        locos.add(loco_info((Locomotive)car))
    }
}
info.put("locomotives", locos)
```